src/cleanfigure.m

```diff
@@ -500,6 +510,46 @@ function simplifyLine(meta, handle, targetResolution)
     removeData(meta, handle, id_remove)
 end
 % =========================================================================
+function limitPrecision(meta, handle, alpha)
+  % Limit the precision of the given data
+
+  % Extract the data from the current line handle.
+  xData = get(handle, 'XData');
+  yData = get(handle, 'YData');
+  if isAxis3D(meta.gca)
+	  zData = get(handle, 'ZData');
+  end
+
+  % Put the data into a matrix
+  if isAxis3D(meta.gca)
+      data  = [xData(:), yData(:), zData(:)];
+  else
+      data  = [xData(:), yData(:)];
+  end
+
+  % Only do something if the data is not empty
+  if isempty(data) || all(all(~isfinite(data)))
+      return
+  end
+
+  % Get the maximal value of the data, only considering finite values
+  maxValue = max(abs(data(isfinite(data))));
+
+  % The least significant bit is proportional to the numerical precision
+  % of the largest number. Scale it with a user defined value alpha
+  eps_range = eps(maxValue) * alpha;
```

Comment:
You might want to get rid of `range` here as well and use something more verbose:
 - `lsb` (that's what I would use, but it's probably quite specific to electrical engineering jargon (DAQ/ADC-related stuff))
 - `precision`
 - ...
since `range` is not the right term and not used anywhere else anymore.